Lib/test/test_ast/test_ast.py

```diff
@@ -814,6 +814,17 @@ def test_repr_large_input_crash(self):
                                     r"Exceeds the limit \(\d+ digits\)"):
             repr(ast.Constant(value=eval(source)))
 
+    def test_unparse_interactive(self):
+        # gh-129598: Fix of ast.unparse() when ast.Interactive contains multiple statements
+        source = "i = 1; 'expr'; raise Exception"
+        self.assertEqual(source, ast.unparse(ast.parse(source, mode='single')))
```

Comment:
You can create a roundtrip local helper to improve readability if you want (or that helper could be toplevel somthat we reuse it in the module elsewhere)